state/cluster/badger/mutator.go

```diff
@@ -22,21 +22,23 @@ import (
 
 type MutableState struct {
 	*State
-	lockManager lockctx.Manager
-	tracer      module.Tracer
-	headers     storage.Headers
-	payloads    storage.ClusterPayloads
+	lockManager      lockctx.Manager
+	tracer           module.Tracer
+	headers          storage.Headers
+	payloads         storage.ClusterPayloads
```

Comment:
maybe just rename those to `clusterHeaders` and `clusterPayloads` for extra clarity